ingest/build-configs/ncbi/defaults/annotations.tsv

```diff
@@ -13,3 +13,5 @@ A/House-Mouse/USA/24-014782-003/2024	division	New Mexico # Based on https://wahi
 A/PETFOOD/USA/24-037325-011/2024	host	Avian # This sequence is likely from cat food, which is turkey https://apps.oregon.gov/oregon-newsroom/OR/ODA/Posts/Post/morasch-meats-voluntary-recall-feline-raw-pet-food-hpai
 A/PETFOOD/USA/24-037325-012/2024	host	Avian # This sequence is likely from cat food, which is turkeyy https://apps.oregon.gov/oregon-newsroom/OR/ODA/Posts/Post/morasch-meats-voluntary-recall-feline-raw-pet-food-hpai
 A/PETFOOD/USA/24-037325-013/2024	host	Avian # This sequence is likely from cat food, which is turkey https://apps.oregon.gov/oregon-newsroom/OR/ODA/Posts/Post/morasch-meats-voluntary-recall-feline-raw-pet-food-hpai
+A/Washington/OR/24-037325-011/2024  host  Avian # This sequence is from Pet Food; like above it's likely turkey
+A/Tiger/CA/24-036321-001-original/2024  host  Nonhuman Mammal # New mammal, Panthera tigris
```

Comment:
For general host annotations such as mapping Panthera tigris to Nonhuman mammal, we usually add these to the [host-map.tsv](https://github.com/nextstrain/avian-flu/blob/master/ingest/build-configs/ncbi/defaults/host-map.tsv). Since I'm going to update the host map anyways, I'll move this over in a separate PR.

Reply:
Updated in https://github.com/nextstrain/avian-flu/pull/118